app/controllers/tags_controller.rb

```diff
@@ -214,6 +214,10 @@ def search
       exclude_has_synonyms: params[:excludeHasSynonyms]
     }
 
+    if filter_params[:limit] && filter_params[:limit].to_i < 0
```

Comment:
I wonder, is that the only spot where we have a `limit` parameter?

Reply:
From a quick grep for `:limit` in app/controllers, I can see about 5 or 6 controllers that may have this issue. I can open a new PR for them and fix them if you'd like me to do that or wait until we see this issue from those controllers in our logs, just let me know what you prefer 😸

Reply:
Let's wait till it hurt.